### DISCUSSION TITLE
Arabic language iOS phone

### DISCUSSION THREAD

**Discussion by alialhammad1992** [community/community#21348] [Mobile]
in Saudi Arabia and add the Arabic GitHub language. Your audience is large there and we want that. We want to GitHub the and make it GitHub the Arabic language

**Comment by aconsuegra** [comment]
Hey @alialhammad1992, thanks for getting in touch and for your request to add support for Arabic language. I'll pass this feedback to our Product team so we can prioritize it accordingly.

**Reply by alialhammad1992** [reply]
https://user-images.githubusercontent.com/104555394/181655404-4d82fead-c6cd-4b80-9116-4a7929015423.MOV


Is it possible to add a direct click in an application?

https://user-images.githubusercontent.com/104555394/181655560-5f23cb21-9f3d-4e9e-8a14-07d2e4521426.MOV

We need to click the same way we click on a browser
